README.md

```diff
@@ -1,2 +1,201 @@
+
+| Characteristic | Specification |
+| :------------: | :-----------: |
+| Operative System | Ubuntu 18.04 |
+| Number of CPUs | 2 |
+| RAM | 4GB |
+| SSD | 80GB | 
+
+Also, we need to make sure the following requirements are met: 
+* Allow incoming connections on port `26656`
+* Have a static IP address
+* Have access to the root user
+
+## 2. Updating
+In order to update the OS so that you can work properly, execute the following commands:
+
+```shell
+# Update Ubuntu  
+apt update && sudo apt upgrade -y
+apt install unzip
+```
+
+## 3. Chain selection
+Before installing the node, please select which chain you would like to connect to 
+
+```shell
+mkdir commercio-chains && cd commercio-chains
+git clone https://github.com/commercionetwork/chains.git .
+cd commercio-<chain-version> # eg. cd commercio-testnet1001 
+```
+
+## 4. Install binaries, genesis file and setup configuration
+
+Download binaries 
+
+```shell
+git init . 
+git remote add origin https://github.com/commercionetwork/commercionetwork.git
+git pull
+git checkout tags/$(cat .data | grep -oP 'Release\s+\K\S+')
+make install
+```
+
+Test if you have the correct binaries version:
+
+```shell
+cnd version
+# Should output the same version written inside the .data file
+```
+
+Setup the validator node name. We will use the same name for node as well as the wallet key:
+
+```shell
+export NODENAME="riccardo"
+cat <<EOF >> ~/.profile
+export NODENAME="$NODENAME"
+EOF
+```
+
+Init the `.cnd` folder with the basic configuration
+
+```shell
+cnd unsafe-reset-all
+
+# If you get a error because .cnd folder is not present don't worry 
+cnd init $NODENAME
+```
+
+Install `genesis.json` file
+
+```shell
+pkill cnd
+rm -rf ~/.cnd/config/genesis.json
+wget https://github.com/commercionetwork/chains/raw/master/commercio-$(cat .data | grep -oP 'Name\s+\K\S+')/genesis.json -P ~/.cnd/config
+```
+
+Change the persistent peers inside `config.toml` file
+
+```shell
+sed -e "s|persistent_peers = \".*\"|persistent_peers = \"$(cat .data | grep -oP 'Persistent peers\s+\K\S+')\"|g" ~/.cnd/config/config.toml > ~/.cnd/config/config.toml.tmp
+mv ~/.cnd/config/config.toml.tmp  ~/.cnd/config/config.toml
+```
+
+Change the seeds inside the `config.toml` file
+```shell
+sed -e "s|seeds = \".*\"|seeds = \"$(cat .data | grep -oP 'Seeds\s+\K\S+')\"|g" ~/.cnd/config/config.toml > ~/.cnd/config/config.toml.tmp
+mv ~/.cnd/config/config.toml.tmp  ~/.cnd/config/config.toml
+```
+
+## 5. Configure the service
+
+```shell
+tee /etc/systemd/system/cnd.service > /dev/null <<EOF  
+[Unit]
+Description=Commercio Node
+After=network-online.target
+
+[Service]
+User=root
+ExecStart=/usr/bin/cnd start
+Restart=always
+RestartSec=3
+LimitNOFILE=4096
+
+[Install]
+WantedBy=multi-user.target
+EOF
+```
+
+Now you can start full node. Enable and try to start service
+
+```shell
+# Start the node  
+systemctl enable cnd  
+systemctl start cnd
+```
+
```

Comment:
**Optional**: you can quick sync with the follow procedure:  
```shell
wget "https://quicksync.commercio.network/commercio-testnet2000.latest.tgz" -P ~/.cnd/
```
Control checksum compare with https://quicksync.commercio.network
```
cd ~/.cnd/
tar -zxf commercio-testnet2000.latest.tgz
```